app/src/main/java/com/drape/util/DateUtils.kt

```
@@ -0,0 +1,95 @@
+        }
+        return format(calendar)
+    }
+
+    /**
+     * Parses a "yyyy-MM-dd" string to Date.
+     * Returns null if parsing fails.
+     */
+    fun parse(dateString: String): Date? {
+        return try {
+            formatter.parse(dateString)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)